lib/OriginalSource.js

```diff
@@ -9,7 +9,7 @@ var SourceMapConsumer = require("source-map").SourceMapConsumer;
 var SourceListMap = require("source-list-map").SourceListMap;
 var Source = require("./Source");
 
-function isSplitter(c) {
+let isSplitter = c => {
```

Comment:
`c => char // (?)`

Reply:
Why `let` ?

Reply:
I have refactored the function signature to arrow functions.
```
function isSplitter(c) => { ... }
```
is equivalent to
```
let isSplitter = c => { ... }
```
This way it is not too verbose and looks elegant. 
I chose to use let because I am treating it as a function expression.
One of the benefits of creating a named function expression is that in case we encountered an error, the stack trace will contain the name of the function, making it easier to find the origin of the error.
I am open to hearing your suggestions. TBH, I may be missing something here.

Reply:
What I meant here is if there is a reason to **not** use `const` instead if `let` :)

Reply:
e.g there is a reassignment to `isSplitter` somewhere

Reply:
No, I don't have any reason. I just wanted to make sure nothing breaks 😄